tools/st2ctl

```diff
@@ -6,7 +6,7 @@ LOGFILE="/tmp/st2_startup.log"
 WEBUI_LOG_FILE="/var/log/st2/st2web.log"
 COMPONENTS="st2actionrunner st2api st2auth st2sensorcontainer st2rulesengine mistral st2web st2resultstracker st2notifier"
 STANCONF="/etc/st2/st2.conf"
-PYTHON=`which python`
```

Comment:
On a related note, we should probably use `#!/usr/bin/env python{2.7}` so it's easier for user to use a different Python interpreter without needing to modify the entry point script or use a symlink.